CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Added
 
+- A general Resources Detector Interface and one implementation FromEnv under folder `sdk/resource`. (#939)
```

Comment:
Can you please instead describe in less technical language? e.g. "Allows Resources to be detected from environment variables"

Reply:
Sure, will do.